workers/tests/unittests/test_api_request_models.py

```diff
@@ -31,17 +36,30 @@ def test_base_indicator_invalid(self):
             request_models.BaseIndicator(
                 name="GhsPopComparisonBuildings", include_svg="foo"
             )
+            request_models.BaseIndicator(
+                name="GhsPopComparisonBuildings", include_html="foo"
+            )
+            request_models.BaseIndicator(
+                name="GhsPopComparisonBuildings", flatten="foo"
+            )
```

Comment:
This test isn't working in general. The test breaks after the first `ValueError` and doesn't check if *every* single test is raising a `ValueError`. To reproduce this, replace the test cases with a valid and an invalid one — the tests shouldn't be valid:

```python
def test_base_indicator_invalid(self):
    with self.assertRaises(ValueError):
        request_models.BaseIndicator()
        request_models.BaseIndicator(name="GhsPopComparisonBuildings")
```

Maybe create another PR for this (or an issue at least) and fix this later on.

Reply:
Ouh. Good catch. Thanks. I made a new commit to fix this.